### ISSUE TITLE
Dashboard with two forest fire warning maps shows empty maps

### ISSUE THREAD

**Issue by Rdataflow** [visualize-admin/visualization-tool#1659]
**Describe the bug** 
A dashboard with two forest fire warning / measaure maps shows emtpy maps

**To Reproduce** Steps to reproduce the behavior:

1. Go to https://test.visualize.admin.ch/v/Lnx2FzF-bdM3?dataSource=Test-uncached
4. See emtpy maps

**Expected behavior** 
Two maps
- 1 map with warnings
- 1 map with measures

**Environment (please complete the following information):**
- Visualize environment and version: [e.g. TEST v4.7.0]

cc @sosiology @bprusinowski

**Comment by sosiology** [comment]
Hi @Rdataflow 
I tried to recreate the dashboard and couldn't reproduce the error. see: https://test.visualize.admin.ch/v/Tv6HttG8CAHX?dataSource=Test-uncached

I noticed in your chart you tried to combine the data sets. Was this intended?
![image](https://github.com/user-attachments/assets/3f57b2b2-08ba-45e6-b3b0-a193d2871e07)

To create a dashboard with 2 charts that each use a separate dataset the function in the tab menu and not in the sidebar needs to be used. 
![image](https://github.com/user-attachments/assets/a84c1415-55ce-4a8a-bad5-98e74655a3a6)
